.gitignore

```diff
@@ -8,6 +8,7 @@
 *.user
 *.userosscache
 *.sln.docstates
+*.sln
```

Comment:
Remove the solution from source control?

Reply:
The solution file is auto generated and will be generated each time when run LaunchVS.ps1